dapp/package.json

```
@@ -13,14 +13,18 @@
     "lint:js:fix": "eslint . --fix",
     "lint:config": "prettier -c '**/*.@(json|yaml|toml)'",
     "lint:config:fix": "prettier -w '**/*.@(json|yaml|toml)'",
-    "test": "vitest"
+    "test": "vitest",
+    "prepare:dev": "mustache config/development.json ledger-manifest-template.json > ledger-manifest-development.json",
+    "prepare:testnet": "mustache config/testnet.json ledger-manifest-template.json > ledger-manifest-testnet.json",
+    "prepare:mainnet": "mustache config/mainnet.json ledger-manifest-template.json > ledger-manifest-mainnet.json",
+    "prepare:all-env": "pnpm prepare:dev && pnpm prepare:testnet && pnpm prepare:mainnet"
```

We could simplify the settings by combining the scripts in one:
```suggestion
    "generate:manifests": "for env in development testnet mainnet; do mustache config/$env.json ledger-manifest-template.json > ledger-manifest-$env.json; done"
```

Additionally we could include the generation in the build step:
```json
"build": "npm run generate:manifests && tsc && vite build",
```